.buildkite/basic/cloudflare-pipeline.yml

```diff
@@ -0,0 +1,16 @@
+steps:
+
+  - group: ":cloudflare: Cloudflare Worker Tests"
+    steps:
+    - label: ":cloudflare: Cloudflare Worker tests"
+      timeout_in_minutes: 40
```

Comment:
nit: it's a longer long for a build machine to be tied up if the tests were to hang for any reason.  Maybe reduce to 15?